lib/strava/api/ratelimit.rb

```diff
@@ -9,6 +9,20 @@ def initialize(response)
         @body = response.body
       end
 
+      def exceeded
+        return false unless limit?
+
+        @exceeded ||= if fifteen_minutes_remaining && fifteen_minutes_remaining <= 0
+                        { fifteen_minutes_remaining: fifteen_minutes_remaining }
+                      elsif total_day_remaining && total_day_remaining <= 0
+                        { total_day_remaining: total_day_remaining }
+                      end
+      end
```

Comment:
In a future PR, I wonder whether we can store response time and also add methods for live how long do I need to wait time. Maybe methods such as `wait_until` or `retry?`, WDYT?

Reply:
It's worth investigating for sure 👍